src/lib/types.ts

```diff
@@ -136,6 +136,39 @@ export interface TestData {
     required: ReqType;
 }
 
+
+/**
+ * (Internal) values for the test scores
+ */
+// eslint-disable-next-line no-shadow
+export enum Score {
+    FAIL = "fail",
+    PASS = "pass",
+    NONAPPLICABLE = "n/a",
```

Comment:
Nit: technically N/A means not applicable. Also not sure what the naming conventions are here, but maybe it should be NOT_APPLICABLE?

Reply:
Yep, done